README.md

```diff
@@ -1,6 +1,10 @@
 # Multiplayer
 Multiplayer mod for RimWorld
 
+This is the new repository from Zetrith's Multiplayer, as he has left and can not be reached we have decided to work on it ourselves.
```

Comment:
This is the official fork from Zetrith's Multiplayer. As he has left and cannot be reached the community is continuing development.